### PR TITLE
added another doc string

### PR DESCRIPTION
**Pull request by josephhalstead**
added a doc string for the multiply function

### CODE REVIEW

File: test_script.py [hunk 1]
```diff
@@ -8,11 +8,16 @@ def add_together(number1, number2):
 
 
 def multiply(number1, number2):
+	"""
+	mutiply two numbers together
+	"""
```
Comment (erikwaskiewicz) [inline test_script.py]: Looks good

File: test_script.py [hunk 2]
```diff
@@ -8,11 +8,16 @@ def add_together(number1, number2):
...
 
 	return number1 * number2
 
 print (add_together(1,5))
 
+print (multiply(5,6))
+
```
Comment (erikwaskiewicz) [inline test_script.py]: nah

Comment (josephhalstead) [reply test_script.py]: changed in new commit

Comment (erikwaskiewicz) [reply test_script.py]: nice one

**Pull request merged**